.env

```diff
@@ -0,0 +1,6 @@
+# Email configuration
+SMTP_SERVER=smtp.gmail.com
+SMTP_PORT=587
+EMAIL_USER=your_email@gmail.com
+EMAIL_PASSWORD=your_password
+NOTIFICATION_EMAIL=bergwolf@gmail.com
```

Comment:
Please do not use a real email address here. It's better use `some_email@gmail.com`.